### PR TITLE
chore: ngc release notes and license

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Requirement for merging https://github.com/determined-ai/environments/pull/235
Adds NGC container license and updates `gen-attributions.py` to account for NVIDIA container license types.
Also adds release note.

## Test Plan

No testing required

## Checklist

- [X] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
MD-270

### CODE REVIEW

**Review by ioga — APPROVED** [review]

File: docs/release-notes/ngc-images.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+**New Features**
+
+-  Add experimental NGC-based images in our environments offerings. These images can be pulled from
+   `https://hub.docker.com/r/determinedai/pytorch-ngc` or
+   `https://hub.docker.com/r/determinedai/tensorflow-ngc`
+
+**Improvements**
+
+-  Remove TF 2.8 images from our offerings. Our default TF 2.11 images are still available for
+   Tensorflow users.
```
Comment (tara-det-ai) [inline docs/release-notes/ngc-images.rst:12]: **New Features**

-  Add experimental NGC-based images in our environments offerings. These images can be pulled from
   `https://hub.docker.com/r/determinedai/pytorch-ngc` or
   `https://hub.docker.com/r/determinedai/tensorflow-ngc`

**Improvements**

-  Remove TF 2.8 images from our offerings. Default TF 2.11 images are still available for TensorFlow users.
